cmd/create/machinepool/nodepool.go

```diff
@@ -51,6 +52,55 @@ func addNodePool(cmd *cobra.Command, clusterKey string, cluster *cmv1.Cluster, r
 		os.Exit(1)
 	}
 
+	// OpenShift version:
+	isVersionSet := cmd.Flags().Changed("version")
+	version := args.version
+	if isVersionSet || interactive.Enabled() {
+		// NodePool will take channel group from the cluster
+		channelGroup := cluster.Version().ChannelGroup()
+		clusterVersion := cluster.Version().RawID()
+		versionList, err := versions.GetVersionList(r, channelGroup, true, true)
+		if err != nil {
+			r.Reporter.Errorf("%s", err)
+			os.Exit(1)
+		}
+
+		// Calculate the minimal version for a new hosted machine pool
+		minVersion, err := versions.GetMinimalHostedMachinePoolVersion(clusterVersion)
+		if err != nil {
+			r.Reporter.Errorf("%s", err)
```

Comment:
Is the error bubbled here is detailed

Reply:
This is an error similar to the "GetVersionList" in case the cluster reports a bad RawID.
The user will have a "Expected a valid OpenShift version:" error in case the version is incorrect.